src/core/network/createNetwork.js

```diff
@@ -72,9 +72,17 @@ export default (config, logger, lifecycle, networkStrategy) => {
             const responseHandlingMessage = expectsResponse
               ? ""
               : " (response will be ignored)";
+            const stringifiedPayload = JSON.stringify(payload);
```

Comment:
Should we pass `stringifiedPayload` to the `networkStrategy` on line 90 ?

Reply:
oof. yes!

Reply:
Done!